Plugin/CppApi/source/ArcGISArView.cpp

```diff
@@ -20,24 +20,25 @@
 #include <QQuickWindow>
 #include <QScreen>
 
+using namespace Esri::ArcGISRuntime;
+using namespace Esri::ArcGISRuntime::Toolkit;
+using namespace Esri::ArcGISRuntime::Toolkit::Internal;
+
 /*!
-  \class Esri::ArcGISRuntime::Toolkit::ArcGISArView
-  \ingroup ArcGISQtToolkitCppApi
+  \class ArcGISArView
+  \ingroup ArcGISQtToolkitAR
   \inmodule ArcGISQtToolkit
   \since Esri::ArcGISRuntime 100.6
   \brief A scene view for displaying ARKit/ARCore features on mobile devices.
 
-  The Augmented Reality toolkit provides support for ARKit for iOS and Android.
-  The Augmented Reality (AR) toolkit component allows quick and easy integration
+  The Augmented Reality (AR) toolkit provides support for ARKit for iOS and
+  ArCore for Android.
+  This toolkit component allows quick and easy integration
   of AR into your application for a variety of scenarios.
 
-  See AR.md for details.
+  See \l {https://github.com/Esri/arcgis-runtime-toolkit-qt/blob/master/Common/AR/Ar.md} {additional details about using the ArcGISArView toolkit component}.
```

Comment:
At one point we were considering renaming Ar.md to Readme.md so it would behave as a landing page. What is the consensus on that?

I think we should do that unless there are good reasons not to.

Reply:
This comment still applies. Now that the file is `README.md`, this reference should be updated.